src/lib/npm.js

```diff
@@ -6,9 +6,6 @@ module.exports = {
     return JSON.parse(data);
   },
   buildInstallCommand: (name, path) => {
-    return 'npm install'
-      .concat(' --no-package-lock')
-      .concat(` --prefix ${path}`)
-      .concat(` ${name}`);
+    return 'npm install'.concat(` --prefix ${path}`).concat(` ${name}`);
```

Comment:
Since we're changing this line anyway:
```suggestion
    return `npm install --prefix ${path} ${name}`;
```